Generator/Package.swift

```diff
@@ -21,7 +21,14 @@ let package = Package(
                 .product(name: "Yams", package: "Yams"),
                 .product(name: "ZIPFoundation", package: "ZIPFoundation"),
             ]
-        )
+        ),
+        .testTarget(
```

Comment:
Can we please also run these on CI? Could be an additional job in the `pull-requests` workflow so they run in parallel.

Reply:
Looks good, thanks!